pkg/storage/unified/sql/data/resource_history_get.sql

```diff
@@ -0,0 +1,21 @@
+SELECT
+  {{ .Ident "resource_version" }},
+  {{ .Ident "namespace" }},
+  {{ .Ident "name" }},
+  {{ .Ident "folder" }},
+  {{ .Ident "value" }}
+FROM {{ .Ident "resource_history" }}
+WHERE 1 = 1
+  AND {{ .Ident "namespace" }} = {{ .Arg .Key.Namespace }}
+  AND {{ .Ident "group" }}     = {{ .Arg .Key.Group }}
+  AND {{ .Ident "resource" }}  = {{ .Arg .Key.Resource }}
+  {{ if .Key.Name }}
+  AND {{ .Ident "name" }}      = {{ .Arg .Key.Name }}
+  {{ end }}
+  {{ if .Trash }}
+  AND {{ .Ident "action" }} = 3
+  {{ end }}
+  {{ if (gt .StartRV 0) }}
+  AND {{ .Ident "resource_version" }} > {{ .Arg .StartRV }}
```

Comment:
Don't we want a Limit/offset here ?

Reply:
It is just like the list command -- no the `StorageBackend` returns an iterator and the the server stops reading when it has returned enough values.